### PR TITLE
Add node local dns latency to load test

### PR DESCRIPTION
**Pull request by marseel**
**What type of PR is this?**
/kind feature
/kind bug

**What this PR does / why we need it**:
Adds measuring of NodeLocalDNSLatency to load test.
Fixes metric name used in NodeLocalDNSLatency.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by wojtek-t** [comment]
/lgtm
/approve

Thanks!

/hold
@prameshj - please cancel hold if that looks fine to you

File: clusterloader2/pkg/measurement/common/nodelocaldns_latency_prometheus.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ import (
 
 const (
 	nodelocaldnsLatencyPrometheusMeasurementName = "NodeLocalDNSLatencyPrometheus"
-	percLatencyQueryTemplate                     = `histogram_quantile(%v, sum(rate(http_request_duration_seconds_bucket[%v])) by (le))`
+	percLatencyQueryTemplate                     = `histogram_quantile(%v, sum(rate(coredns_health_request_duration_seconds_bucket[%v])) by (le))`
```
Comment (prameshj) [inline clusterloader2/pkg/measurement/common/nodelocaldns_latency_prometheus.go:32]: The metric name is "kubernetes.io/internal/nodes/coredns/dns_request_duration_seconds". Should we use "coredns_dns_request_duration_seconds"?

Comment (marseel) [reply clusterloader2/pkg/measurement/common/nodelocaldns_latency_prometheus.go]: I've changed it to "coredns_dns_request_duration_seconds"
Yesterday, we discovered that it was not working due to #1652 
I am running one more test to make sure it works with current setup.

Comment (marseel) [reply clusterloader2/pkg/measurement/common/nodelocaldns_latency_prometheus.go]: I think we can merge as it is right now, but from my initial test I can see that some requests end up in bucket inf+, which results in 99 percentile to be ~8s (two highest buckets are 8s and +inf) so this test fails.

Comment (prameshj) [reply clusterloader2/pkg/measurement/common/nodelocaldns_latency_prometheus.go]: okay, thanks for trying it out. The threshold value can be overridden by commandline if needed and we can make tweaks to kube-dns replica count to measure how much of a scale-up is needed to achieve 5s. Do you know what the 50th percentile and 90th percentile values were in your test?

**Comment by wojtek-t** [comment]
@marseel - please squash commits and it seems we're good to go

**Comment by marseel** [comment]
I've squashed it, although I think two commits made sense before - one with bugfix, one with feature ;)

**Comment by wojtek-t** [comment]
/lgtm
/hold cancel

> I've squashed it, although I think two commits made sense before - one with bugfix, one with feature ;)

Yeah - that's true. That made sense too (didn't think about that)

**Pull request merged**